infrastructure/terragrunt/aws/ecs/ecs.tf

```diff
@@ -52,6 +52,9 @@ data "template_file" "wordpress_container_definition" {
     S3_UPLOADS_KEY               = aws_secretsmanager_secret_version.s3_uploads_key.arn
     S3_UPLOADS_SECRET            = aws_secretsmanager_secret_version.s3_uploads_secret.arn
     S3_UPLOADS_BUCKET_URL        = "https://${var.domain_name}"
+    C3_AWS_ACCESS_KEY_ID         = aws_secretsmanager_secret_version.c3_aws_access_key_id.arn
+    C3_AWS_SECRET_ACCESS_KEY     = aws_secretsmanager_secret_version.c3_aws_secret_access_key.arn
+    C3_DISTRIBUTION_ID           = var.c3_distribution_id
```

Comment:
Does this need to be in the `container-definitions/wordpress.json.tmpl` ECS container def as well?

Reply:
yup